### ISSUE TITLE
Cannot play on Any Airplay Device

### ISSUE THREAD

**Issue by BenDataAnalyst** [owntone/owntone-server#1740]
Hello,

I've spent two evenings on this issue, trying to read the former issues on GitHub and also on other forums, but no luck so far.

Here are the details:

- I am running OwnTone in a Proxmox container (unprivileged). Debian 12 Bookworm.
- I've installed the latest OwnTone version. 
- Here is the log file in debug mode: [owntone.log](https://github.com/owntone/owntone-server/files/14817696/owntone.log)
- The configuration file is adjusted as follows. I've removed most of the commented lines.

# Enable/disable IPv6
ipv6 = false // I have tried "yes", "no". But I can try again. 

# AirPlay per device settings # I have done the same for another Airplay device
airplay "HomePod mini" {
# present. The speaker will remain until restart of OwnTone.
#       permanent = false
# Disable AirPlay 1 (RAOP)
raop_disable = true
# Name used in the speaker list, overrides name from the device
#       nickname = "My speaker name"
}

**Comment by ejurgensen** [comment]
Are the Airplay speakers shown in the speaker list? (I didn't read the log since it's rather lengthy)

I guess in any case the most likely cause is that you are running in a container. 

Btw I would suggest dropping the per speaker config, it shouldn't be necessary.

**Comment by BenDataAnalyst** [comment]
Yes, the Airplay speakers are showing in the speaker list. When I try to activate one of them (to play a song through one of them), I see the handshake between the AirPlay speaker (HomePod Mini) in the debug log, but then nothing happens. Should I do something with the firewall and ports?
Is running in a container not recommended?
What is the best hardware setup? A Pi?

**Comment by ejurgensen** [comment]
> Should I do something with the firewall and ports?

Airplay opens connections both ways and using random ports, so you should try disabling any firewall. If you have another Airplay speaker, then give that a shot too. While HomePod Mini should work, I think it can be set in the Home app to some grouping mode that can cause difficulties (my memory is a bit vague on that - I don't have one myself).

> Is running in a container not recommended? What is the best hardware setup? A Pi? Should I try different OS, such as FreeBSD?

People run it on all sorts of platforms, but the easiest is probably a Pi, where you can just add the repo (see the Install instructions). @hacketiwack has also made it available via Docker, if you prefer a container, so that should also be quite easy.

**Comment by Porco-Rosso** [comment]
Just chiming in: I am running owntone in an LXC container under Proxmox. Didn't have to do any thing, using the default bridged network interface. Avahi installed, without any special work besides apt install.

**Comment by marioscube** [comment]
I today made a post about owntone not working with my homepod mini (airplay).

It used to work and the only thing I recently changed was this setting in the conf file:
ipv6 = no 
to 
ipv6 = yes

Strangely enough is has worked (I think) but now it does not.

I reverted the setting to **ipv6 = no** and now it all seems to work again.

**Comment by Ben-Sky** [comment]
I close this issue. I will follow up later.

**Comment by Ben-Sky** [comment]
I close this issue. I will follow up later.
